docker/pdo_services_base.dockerfile

```diff
@@ -36,6 +36,8 @@ RUN echo "deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu ${U
     --no-install-recommends \
     libsgx-urts \
     libsgx-uae-service \
+    libsgx-dcap-ql-dev \
+    libsgx-dcap-quote-verify-dev \
  && apt-get clean \
```

Comment:
can we move all of the dcap related installs to one place and then comment about the purpose of the installs

Reply:
done - per our conversations